python/pylintrc

```diff
@@ -45,7 +45,7 @@ extension-pkg-allow-list=
 
 # Minimum supported python version
 # CHANGED
-py-version = 3.4
+py-version = 3.6
```

Comment:
It'd be great if we could get autoconf to substitute this for us, but it looks like it only knows the found version of python, not the minimum version we asked for.

Reply:
We could put it in a variable and AC_SUBST() it. Do you really want a pylintrc.in though?

Reply:
On the one hand, I want less stuff to have to remember to change.  On the other hand, I don't really want autoconf to do more.

Reply:
You could just add a note on T1